common/src/main/java/com/wynntils/wc/utils/EntityUtils.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wc.utils;
+
+import com.wynntils.mc.utils.McUtils;
+import java.util.List;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.entity.animal.horse.AbstractHorse;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.phys.AABB;
+
+public class EntityUtils {
+
+    public static boolean isPlayersHorse(AbstractHorse horse, Player player) {
+        if (horse == null) return false;
+
+        String playerName = player.getName().getString();
+        String defaultName = "§f" + playerName + "§7" + "'s horse";
+        String customNameSuffix = "§7" + " [" + playerName + "]";
+
+        Component horseName = horse.getCustomName();
+        if (horseName == null) return false;
+
+        return defaultName.equals(horseName.getString())
+                || horseName.getString().endsWith(customNameSuffix);
+    }
+
+    public static AbstractHorse searchForHorseNearby(int searchRadius, Player player) {
```

Comment:
i think the Player argument can be removed in favor of calling `McUtils.mc().player` inside the method, like you did for all the other util methods here - we only ever need to search for the player's own horse, after all

Reply:
Right now, yes. Maybe in the future we'd want to search for other players horses. 

But sure, I'm the one who usually argues that designing for future use is futile and bound to be wrong. :) If you really want, I can remove the player argument.

Reply:
Yeah. If in the future we decide to do something like to glow the horses of players as well we can change that later.

Reply:
in this case specifically it's hard to foresee any use in searching for a different player's horse, since any horse that's not your own is essentially useless